src/main/resources/messages/TranslationBundle_zh.properties

```diff
@@ -162,4 +162,21 @@ support.donate=如果您喜欢这个插件，请考虑捐赠以维持插件的
 support.donate.alipay=支付宝
 support.donate.wechat=微信支付
 translate.documentation.disable=关闭快速文档实时翻译
-translate.documentation.limitHint=内容长度超出限制，已跳过翻译。
\ No newline at end of file
+translate.documentation.limitHint=内容长度超出限制，已跳过翻译。
```

Comment:
@YiiGuxing Please check that suggested messages are ok. They were translated using the Translation plugin itself 😎.

Reply:
I will, thanks!